client/src/pages/Login.js

```diff
@@ -119,7 +141,7 @@ const Login = ({ isLoginForm }) => {
 
   return (
     <div className="text-center">
-      <h1>Welcome</h1>
+      <Title>{isLoginForm ? "Sign In" : "Sign Up"}</Title>
       <form id="login-password" method="POST">
         <InputWrapper>
           <Label style={StyleLabel} label="E-mail" />
```

Comment:
I think the input labels need a bit more styling to match Figma

Current implementation:
<img width="97" alt="Screen Shot 2020-05-03 at 9 22 01 PM" src="https://user-images.githubusercontent.com/42228386/80936001-22f67c80-8d84-11ea-8cdb-43dba9ec3df1.png">

Figma:
<img width="73" alt="Screen Shot 2020-05-03 at 9 22 35 PM" src="https://user-images.githubusercontent.com/42228386/80936030-37d31000-8d84-11ea-9cf7-fde794b9a09e.png">

<img width="196" alt="Screen Shot 2020-05-03 at 9 24 15 PM" src="https://user-images.githubusercontent.com/42228386/80936086-736dda00-8d84-11ea-8291-75edba0f8cba.png">